IPython/html/static/edit/js/main.js

```diff
@@ -75,4 +77,14 @@ require([
         }
     };
 
+    // Make sure the codemirror editor is sized appropriatley.
+    var _handle_resize = function() {
+        var header = $('#header');
+        var padding = header.outerHeight(true) - header.innerHeight();
+        $('div.CodeMirror').height(window.innerHeight - header.height() - 2*padding);
+    };
+    window.onresize = _handle_resize;
+
+    // On document ready, resize codemirror.
+    $(_handle_resize);
```

Comment:
I hadn't even realised that the jQuery constructor took a function, but it's there [in the docs](http://api.jquery.com/jQuery/#jQuery3)...